### PR TITLE
LopezCarlos-Topic0

### PR DESCRIPTION
**Pull request by glb-charloz**

### CODE REVIEW

File: 00-HTML-CSS-basics/CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#header, #footer, #aside, #nave {
+   background-color: #696969;
+ }
+
+/* --------------------------------------------------------------------------*/
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/CSS/style.css]: This comment are not needed

File: 00-HTML-CSS-basics/Practica0-2-3/CSS/normalize.css [hunk 2]
```diff
@@ -0,0 +1,461 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/Practica0-2-3/CSS/normalize.css:1]: You could have one reset file and use it in several places

File: 00-HTML-CSS-basics/Practica1-2.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <td>1200</td>
+      </tr>
+      <tr>
+        <td>LitoralGas</td>
+        <td>September</td>
+        <td>paid out</td>
+        <td>700</td>
+      </tr>
+      <tr>
+        <td>Aguas</td>
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/Practica1-2.html:28]: Avoid using spanish :p

File: 00-HTML-CSS-basics/Practica1-4.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <title>.:SignUp:.</title>
+ </head>
+ <body>
+   <div>
+      <form>
+        <h1>Please sign in</h1>
+        <label for="inputFirstname" >First name:</label>
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/Practica1-4.html]: `for` attribute should match with the `id` of the input. Also the label should wrap the input.

File: 00-HTML-CSS-basics/Practica1-4.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <input name="email" id="mail"  required="" pattern="^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$">
+        <div>
+        <label for="inputBirthday" >Bithday:</label>
+        <input name="Birthday" type="date" required="">
+        <div>
+        <label for="inputSport" >Select your favourite Sport:</label>
+        <select>
+          <option value="basquet">Basquet</option>
+          <option value="tennis">Tennis</option>
+          <option value="futball">Futball</option>
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/Practica1-4.html]: Minor: it is football

**Review by afiorenza — CHANGES_REQUESTED** [review]
Just a few comments. Well done!

File: 00-HTML-CSS-basics/Practica1-2.html (hunk 3 above)
Comment (glb-charloz) [reply 00-HTML-CSS-basics/Practica1-2.html:28]: Whit "aguas" i refer to "aguas provinciales" ( the company), its not clear, sorry.

**Review by afiorenza — APPROVED** [review]

**Pull request merged**